src/Agent.Listener/LocalRunner.cs

```diff
@@ -261,36 +279,36 @@ public sealed override void Initialize(IHostContext hostContext)
                     builder.Append($@"
       ""system"": ""build"",
       ""system.collectionId"": ""00000000-0000-0000-0000-000000000000"",
-      ""system.teamProject"": ""gitTest"",
-      ""system.teamProjectId"": ""00000000-0000-0000-0000-000000000000"",
+      ""system.culture"": ""en-US"",
```

Comment:
I sorted all of these variables. The actual changes here are all the string interpolation to set the source version, etc.